tests/test_oc4ids_tranforms.py

```diff
@@ -0,0 +1,1332 @@
+    assert output["contractingProcesses"][2]["summary"]["status"] == "closed"
+    assert output["contractingProcesses"][3]["summary"]["status"] == "closed"
+    assert output["contractingProcesses"][4]["summary"]["status"] != "closed"
+    assert output["contractingProcesses"][5]["summary"]["status"] != "closed"
+    assert output["contractingProcesses"][6]["summary"]["status"] != "closed"
+
+
+def test_procurment_process():
+
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "tender": {"procurementMethod": "method", "procurementMethodDetails": "details"},
+        }
+    ]
+
+    output = transforms._run_transforms(
+        copy.deepcopy(releases),
+        "1",
+        transforms=[transforms.contracting_process_setup, transforms.procurement_process],
+    )
+
+    assert output["contractingProcesses"][0]["summary"]["tender"] == releases[0]["tender"]
+
+
+def test_number_of_tenderers():
+
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "tender": {"numberOfTenderers": 123},
+        }
+    ]
+
+    output = transforms._run_transforms(
+        copy.deepcopy(releases),
+        "1",
+        transforms=[transforms.contracting_process_setup, transforms.number_of_tenderers],
+    )
+
+    assert output["contractingProcesses"][0]["summary"]["tender"]["numberOfTenderers"] == 123
+
+
+def test_location():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "planning": {"project": {"locations": [{"description": "Mars"}]}},
+        }
+    ]
+
+    output = transforms._run_transforms(copy.deepcopy(releases), "1", transforms=[transforms.location],)
+
+    assert output["locations"] == [{"description": "Mars"}]
+
+
+def test_location_from_item_location():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "tender": {
+                "items": [
+                    {
+                        "id": "item1",
+                        "deliveryLocation": {
+                            "geometry": {"type": "Point", "coordinates": [51.751944, -1.257778]},
+                            "uri": "http://www.geonames.org/2640729/oxford.html",
+                        },
+                    }
+                ],
+            },
+        }
+    ]
+
+    output = transforms._run_transforms(
+        copy.deepcopy(releases), "1", transforms=[transforms.location, transforms.location_from_items],
+    )
+    assert output["locations"] == [releases[0]["tender"]["items"][0]["deliveryLocation"]]
+
+
+def test_location_from_delivery_address():
```

Comment:
Should this test also cover the following inputs?

* A contracting process with multiple items, each with different `deliveryAddress`s
* Multiple contracting processes, each with different `tender.items.deliveryAddress`